ecommerce/models.py

```diff
@@ -373,6 +373,9 @@ def fulfill(self, payment_data):
             keep_failed_enrollments=True,
         )
 
+        # send the receipt emails
+        send_ecommerce_order_receipt(self)
```

Comment:
It would be a good measure to put this in a celery task so we have a guarantee this email will eventually send even if mailchimp fails.